### DISCUSSION TITLE
Pressing "Esc" on a nested dialog close all the dialogs

### DISCUSSION THREAD

**Discussion by GutoFrr** [primefaces/community#2927] [PrimeVue]
Whenever I press the escape key on a nested dialog, all the parent dialogs close at the same time, instead of only the current child dialog being closed. I've recorded a video to show how this happens.

https://github.com/user-attachments/assets/6fa98440-7289-4d05-adcd-a239583c9615

Sorry about the censorship.

**Comment by j0rgedev** [comment]
Hey @GutoFrr. It seems like that's the way nested dialogs work in [PrimeVue](https://primevue.org/dynamicdialog/#example)
